### DISCUSSION TITLE
thanks

### DISCUSSION THREAD

**Discussion by houdini68** [roelderickx/connective-plugin-linux#13] [General]
@roelderickx,
I wanted to thank you for the integration of 'Connective signing extension' into Firefox (in my case).
I'm shocked that we have to use Github in order to install/integrate this plugin in a browser on Linux (IMHO a shame!).
The readme + readme (found into test) described correctly what to install, how to install the plugin into the browser (Firefox) and how to test.
My wife created her ITSME account from Ubuntu 20.04 this morning and it was successful.
I big big thanks for your work!
PS : I did some screen dumps of the installation/integration on Ubuntu if it can help somebody else.

**Comment by roelderickx** [comment]
Great to hear it works for you! I am a bit surprised you had succes with Itsme, some other people found out Itsme is using a different and incompatible Connective plugin. See https://github.com/roelderickx/connective-plugin-linux/issues/11#issuecomment-1029364411.

Anyway, I agree it is a shame we need to create such a hacky solution. Especially since Connective advertised for a long time they intended to support Linux. I contacted Doccle support around december 2020 to ask for a Linux version and they too promised me it would be implemented, but both Connective and Doccle have since removed this promise from their website.

Disappointing, and from what I've read in several issues also extremely frustrating.

**Reply by houdini68** [reply]
I confirm we did a ITSME registration. The browser didn't complain the plugin was not installed.
